tide_site.module

```diff
@@ -623,6 +587,20 @@ function tide_site_node_update($node) {
     $alias_helper = \Drupal::service('tide_site.alias_storage_helper');
     $alias_helper->regenerateNodeSiteAliases($node);
   }
+  $old_node = $node->original;
```

Comment:
@vincent-gao `hook_node_update` is not invoked when the node is first saved. You need the same logic for `hook_node_insert`.

Reply:
@sonnykt . something will be invoked by somehow. so the logic is right, no need for `hook_node_insert`

Reply:
@vincent-gao thanks for pointing out that previous implementation was all wrapped in `if (!$node->isNew())` which means that the logic is only valid for updates.